heron/metricsmgr/src/java/BUILD

```diff
@@ -16,11 +16,12 @@ java_library(
         "//heron/proto:proto_metrics_java",
         "//heron/proto:proto_tmaster_java",
         "//heron/metricsmgr/src/thrift:thrift_scribe_java",
+        "//third_party/java:guava",
```

Comment:
Can you add a comment here to indicate this library is used only for websink, since we would later split the all these sinks from the spi parts.